BadHabbit/BadHabbit/SupportFiles/Design Constants/StartButton.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  MainButton.swift
+//  BadHabbit
+//
+//  Created by Halina Kurylchykava on 23.07.23.
+//
+
+import Foundation
+
+let buttonCornerRadius: CGFloat = 12
```

Comment:
Please, never define variables in global scope

Reply:
changed